src/Command/DumpEnvCommand.php

```diff
@@ -42,6 +42,7 @@ protected function configure()
                 new InputArgument('env', InputArgument::OPTIONAL, 'The application environment to dump .env files for - e.g. "prod".'),
             ])
             ->addOption('empty', null, InputOption::VALUE_NONE, 'Ignore the content of .env files')
+            ->addOption('env-file', null, InputOption::VALUE_OPTIONAL, 'The name of the .env files to be dumped, relative to the project root dir.', '.env')
```

Comment:
this should be `InputOption::VALUE_REQUIRED`. There is no reason to pass `--env-file` without a value in the option.

Reply:
Fixed